include/platform/settings.h

```diff
@@ -36,7 +36,7 @@
 #define OT_PLATFORM_SETTINGS_H 1
 
 #include <stdint.h>
```

Comment:
Can we remove the `stdint.h` include, since `openthread-types.h` already includes it?